connectivity/FEATURE_BLE/source/cordio/stack_adaptation/hci_tr.c

```diff
@@ -170,8 +170,18 @@ void hciTrSerialRxIncoming(uint8_t *pBuf, uint8_t len)
           hdrLen = HCI_EVT_HDR_LEN;
           break;
         default:
-          /* invalid packet type */
+          /**
+           * invalid packet type 
+           * 
+           * fix: Simply employing WSF_ASSERT in the event of 
+           * an "invalid packet type" is not reasonable. 
+           * Instead, it is advisable to discard this data packet, 
+           * exit the packet processing function, 
+           * and adjust the stateRx back to HCI_RX_STATE_IDLE.
+          */
+          stateRx = HCI_RX_STATE_IDLE;
           WSF_ASSERT(0);
+          return;
```

Comment:
break is not needed anymore afer we added return. should we remove it?

Reply:
Yes

Reply:
lets remove it

Reply:
Done